### PR TITLE
enhancement:added meta tags and images

### PR DESCRIPTION
**Pull request by GayathriHG**
added meta tags ,images and og  SEO.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Wish your friends and family with this New Year card with your names on it !!"/>
+  <meta name="keywords" content="New Year,2024,Happy New Year,Party "/>
+  <meta name="author" content="Anbu Selvan"/>
+  <meta property="og:title" content=" New Year Greeting App" />
+  <meta name="og:description" content="Wish everyone a Happy New Year with names on it. Give it a try !!"/>
+  <meta
+  property="og:image"
+  content ="https://i.pinimg.com/564x/1e/f2/d9/1ef2d9a88ec3a8abbaf1b60b8ae23b7d.jpg"
```
Comment (anburocky3) [inline index.html]: You should reference it with the local image la?

Comment (anburocky3) [reply index.html]: @GayathriHG

**Review by anburocky3 — CHANGES_REQUESTED** [review]
You need to address it to approve.

File: index.html (hunk 1 above)
Comment (GayathriHG) [reply index.html]: I will have this path, is it ok?

1. Create `public/img`, put your file on the `img` folder.
2. Reference it `/festival-greeting-app/img/festival-greeting-app.jpg`

**Review by anburocky3 — APPROVED** [review]

**Comment by anburocky3** [comment]
Thanks @GayathriHG

**Pull request merged**